YDS-Storybook/SwiftUI/Atom/LabelPageView.swift

```diff
@@ -12,35 +12,39 @@ import YDS_SwiftUI
     let title: String = "Label"
 
     @State var text: String? = "Label"
-    @State var int: Int = 0
     @State var typoStyleSelectedIndex = 0
+    @State var lineLimit: Int? = nil
```

Comment:
옵셔널로 선언하면 nil로 초기화를 안해줘도 될 것 같아요
SwiftLint에서도 경고를 띄우네요

```suggestion
    @State var lineLimit: Int?
```

Reply:
넵 반영하겠습니다!